SU-HKKU/cardinal_service_profile
# CSD-482 Add summary field for opportunity lists

# READY FOR REVIEW

# Summary
- Added summary field for opportunities lists
- @rebeccahongsf feel free to add styles in this PR or in another one.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout this branch
1. `blt drupal:sync --site=cardinalservice`
1. `drush @cardinalservice.local mim solo_opportunities --update`
1. view the opportunities list page to see the summaries


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
